case-lib/pipeline.sh

```diff
@@ -28,18 +24,23 @@ func_pipeline_export()
         done
     fi
 
-    local opt="-f \"$filter\" -b \"$ignore\""
-    local cmd="sof-tplgreader.py $tplg_path $opt -s $sofcard -e"
+    local opt=""
+    # acquire filter option
+    [[ "$2" ]] && opt="-f '$2' "
+    [[ "$ignore" ]] && opt="-b '$ignore' "
+    [[ "$SOFCARD" ]] && opt="-s $SOFCARD"
+
+    local -a pipeline_lst
+    local cmd="sof-tplgreader.py $tplg_path $opt -e" line=""
```

Comment:
This looks like real change, really not just fixing a warning. Explain this in a different commit or even better in a different PR.